requirements.txt

```diff
@@ -1,13 +1,15 @@
 PyYAML<4.3
 gunicorn<20.0.0
 matplotlib
+multi-model-server==1.0.8.1
```

Comment:
The most recent release of MMS is multi-model-server 1.1.0. We have done a lot of testing against this version. Would highly recommend to upgrade to this. 

https://pypi.org/project/multi-model-server/1.1.0/